website/src/pages/index.js

```diff
@@ -50,6 +50,11 @@ function HomepageHeader() {
               to="/puzzles/day6">
             Day 6
           </Link>
```

Comment:
This one (for day 6) should become `button--secondary`

Reply:
Ach, that's why it was red :S